src/mapeo-project.js

```diff
@@ -206,51 +207,61 @@ export class MapeoProject extends TypedEmitter {
         dataStore: this.#dataStores.data,
         table: observationTable,
         db,
+        translation: this.$translation,
       }),
       track: new DataType({
         dataStore: this.#dataStores.data,
         table: trackTable,
         db,
+        translation: this.$translation,
       }),
       preset: new DataType({
         dataStore: this.#dataStores.config,
         table: presetTable,
         db,
+        translation: this.$translation,
       }),
       field: new DataType({
         dataStore: this.#dataStores.config,
         table: fieldTable,
         db,
+        translation: this.$translation,
       }),
       projectSettings: new DataType({
         dataStore: this.#dataStores.config,
         table: projectSettingsTable,
         db: sharedDb,
+        translation: this.$translation,
       }),
       coreOwnership: new DataType({
         dataStore: this.#dataStores.auth,
         table: coreOwnershipTable,
         db,
+        translation: this.$translation,
       }),
       role: new DataType({
         dataStore: this.#dataStores.auth,
         table: roleTable,
         db,
+        translation: this.$translation,
       }),
       deviceInfo: new DataType({
         dataStore: this.#dataStores.config,
         table: deviceInfoTable,
         db,
+        translation: this.$translation,
       }),
       icon: new DataType({
         dataStore: this.#dataStores.config,
         table: iconTable,
         db,
+        translation: this.$translation,
       }),
       translation: new DataType({
         dataStore: this.#dataStores.config,
         table: translationTable,
         db,
+        translation: this.$translation,
```

Comment:
Can we prevent people from using translations for translations, which would create an infinite loop?

Reply:
How can be do that? could be make `translationApi` a noop if inside a `DataType` of translation? Or check every call to the api and avoid it if in translation `DataType` ?

Reply:
Done in #604.

Reply:
merged that!